projects/laji/src/app/shared-modules/document-viewer/document-annotation/document-annotation.component.ts

```diff
@@ -243,6 +243,7 @@ export class DocumentAnnotationComponent implements AfterViewInit, OnChanges, On
             return hasHighlight;
           }
           hasHighlight = this.shouldOnlyShowHighlighted(subLevel, highlight);
+          return undefined;
```

Comment:
`forEach` function shouldn't return anything so maybe this could be refactored like this
```
if (!hasHighlight) {
    hasHighlight = this.shouldOnlyShowHighlighted(subLevel, highlight);
}
```